packages/flutter/lib/src/widgets/sliver.dart

```diff
@@ -746,6 +746,21 @@ class SliverChildListDelegate extends SliverChildDelegate {
 }
 
 /// A base class for sliver that have [KeepAlive] children.
+/// It is an abstart class.
+///
+///
+/// See also:
+///
+///  * [KeepAlive], used with SliverWithKeepAliveWidgets as base parent,
+///     this widgets wraps around the child which needs to   
+///     be kept alive. Though it is not need when we use delegates.
```

Comment:
I would shorten the description of [KeepAlive], because in the [KeepAlive] docs, it mentions that "This widget is rarely used directly." So, while it's nice to have a reference to [KeepAlive], its elaboration on this doc doesn't have to be too complex.

```suggestion
///  * [KeepAlive], which marks whether its child widget should be kept alive.
```